### PR TITLE
Telemetry data adjustment

### PR DESCRIPTION
**Pull request by ousmaneo**
This PR adjusts the telemetry events properties.

/jenkins-pr-deps Graylog2/graylog-plugin-enterprise#5141

/nocl

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx [hunk 1]
```diff
@@ -72,6 +74,14 @@ class EventNotificationForm extends React.Component {
...
   handleSubmit = (event) => {
+    const { sendTelemetry } = this.props;
+
+    sendTelemetry('form_submit', {
+      app_pathname: 'events',
+      app_section: 'event-notification',
+      app_action_value: 'save',
+    });
+
     const { notification, onSubmit } = this.props;
```
Comment (gally47) [inline graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx:85]: We are destructuring the props object twice here

File: graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx [hunk 2]
```diff
@@ -93,13 +103,30 @@ class EventNotificationForm extends React.Component {
...
   handleTestTrigger = () => {
+    const { sendTelemetry } = this.props;
+
+    sendTelemetry('input_value_change', {
+      app_pathname: 'events',
+      app_section: 'event-notification',
+      app_action_value: 'notification-test',
+    });
+
     const { notification, onTest } = this.props;
```
Comment (gally47) [inline graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx:130]: same here

**Review by gally47 — APPROVED** [review]
Frontend LGTM! there is one small nitpick

**Review by moesterheld — APPROVED** [review]
Backend part lgtm

File: graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx (hunk 2 above)
Comment (ousmaneo) [reply graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx]: Good findings. thank you

**Pull request merged**